readme.md

```diff
@@ -1,16 +1,16 @@
 [![Build status](https://github.com/koaning/scikit-lego/workflows/Unit%20Tests/badge.svg)](https://github.com/{github_id}/{repository}/workflows/{workflow_name}/badge.svg)
```

Comment:
Not sure if `Build status` is still relevant, I couldn't find a passing status though